examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -4280,6 +4280,40 @@ provisional cluster EnergyEvse = 153 {
   timed command ClearTargets(): DefaultSuccess = 7;
 }
 
+/** This cluster provides an interface to specify preferences for how devices should consume energy. */
+cluster EnergyPreference = 155 {
+  revision 1; // NOTE: Default/not specifically set
```

Comment:
This should be set in the XML. If matter says "default" this means it was not explicitly set.

Reply:
Ok, I'll fix the XML in the new PR